test/test_config_converter.py

```diff
@@ -0,0 +1,47 @@
+"""
+File to run tests for the config_converter algorithm
+
+Usage: python3 -m pytest
+Note: Run this file from the parent directory (outside test folder)
+"""
+
+import subprocess
+
+
+def read_file(path):
+    """Reads file contents at path."""
+    with open(path, 'rt') as f:
+        return f.read()
+
+
+def check_equality(config_name):
```

Comment:
Instead of creating a file in the directory in this repo, I recommend creating file under temp directory (https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)

The reason is that if the test is aborted for some reason, the created file couldn't be cleaned up. Usually if it's under temporary directory of the platform, the created temporary files can be cleaned up.

Reply:
This is a really cool module I didn't know existed! I'll update the tests to use this.